protocols/test_protocol.py

```
@@ -1,8 +1,10 @@
 # Default protocol: test_protocol
 # Auto-generated by protocol loader
 
+
 def task():
```

Ruff (0.11.9)

Missing return type annotation for public function `task`

(ANN201)